Form-Controls/index.html

```diff
@@ -1,27 +1,79 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
-  </head>
-  <body>
-    <header>
-      <h1>Product Pick</h1>
-    </header>
-    <main>
-      <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
-      </form>
-    </main>
-    <footer>
-      <!-- change to your name-->
-      <h2>By HOMEWORK SOLUTION</h2>
-    </footer>
-  </body>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>T-Shirt Order Form</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+  <form aria-label="T-Shirt Order Form">
+    <div>
+      <label for="name">Name</label>
+      <input type="text" name="name" id="name" required />
+    </div>
+
+    <div>
+      <label for="password">Password</label>
+      <input type="password" name="password" id="password" required />
+    </div>
+
+    <div>
+      <label for="email">Email</label>
+      <input type="email" name="email" id="email" required />
+    </div>
+
+    <fieldset>
+      <legend>T-Shirt Size</legend>
+      <div>
+        <input type="checkbox" name="size" id="xs" value="XS" />
+        <label for="xs">X-Small</label>
+      </div>
+      <div>
+        <input type="checkbox" name="size" id="small" value="S" />
+        <label for="small">Small</label>
+      </div>
+      <div>
+        <input type="checkbox" name="size" id="medium" value="M" />
+        <label for="medium">Medium</label>
+      </div>
+      <div>
+        <input type="checkbox" name="size" id="large" value="L" />
+        <label for="large">Large</label>
+      </div>
+      <div>
+        <input type="checkbox" name="size" id="xl" value="XL" />
+        <label for="xl">X-Large</label>
+      </div>
+      <div>
+        <input type="checkbox" name="size" id="xxl" value="XXL" />
+        <label for="xxl">XX-Large</label>
+      </div>
+    </fieldset>
+
+    <div>
+      <label for="Colours">T-Shirt Colour</label>
+      <select name="Colours" id="Colours" required>
```

Comment:
Can you review the exercise and see how many t-shirt colours should be made available to the customer.